org.eclipse.xtext.common.types/src/org/eclipse/xtext/common/types/util/DeprecationUtil.java

```diff
@@ -23,7 +23,8 @@
 public class DeprecationUtil {
 	public static boolean isDeprecated(JvmAnnotationTarget jvmAnnotationTarget) {
 		if (jvmAnnotationTarget instanceof JvmMember) {
-			return isDeprecatedMember((JvmMember) jvmAnnotationTarget);
+			return isDeprecatedMember((JvmMember) jvmAnnotationTarget) 
```

Comment:
The semantics is now the same as for #isTransitivelyDeprecated. Why doesn't the caller use that one instead?